pkg/imagestore/imagestore.go

```diff
@@ -268,17 +273,45 @@ func isoFileName(imageType, openshiftVersion, version, arch string) string {
 	return fmt.Sprintf("rhcos-%s-%s-%s-%s.iso", imageType, openshiftVersion, version, arch)
 }
 
-func buildRootfsURL(scheme, host, arch, version string) string {
+func buildRootfsURL(scheme, host, arch, version string) (string, error) {
+	if strings.Contains(host, ":") && !strings.Contains(host, "http") {
+		// url can't parse host with colon when scheme is missing
+		host = fmt.Sprintf("%s://%s", scheme, host)
```

Comment:
This assumes scheme is passed.

Reply:
Yeah, so in this case we'll get 'missing protocol scheme' error, which should be fine for now I guess.